packages/authorization-process/test/getClientUsers.test.ts

```diff
@@ -0,0 +1,58 @@
+import { getMockClient } from "pagopa-interop-commons-test/src/testUtils.js";
+import { Client, generateId, unsafeBrandId } from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  clientNotFound,
+  organizationNotAllowedOnClient,
+} from "../src/model/domain/errors.js";
+import { addOneClient, authorizationService } from "./utils.js";
+
+describe("getClientUsers", async () => {
+  const consumerId = generateId();
+  const userIds1 = generateId();
+  const userIds2 = generateId();
```

Comment:
```suggestion
  const consumerId = generateId<TenantId>();
  const userId1 = generateId<UserId>();
  const userId2 = generateId<UserId>();
```
Giving the type can spare you to do the `unsafeBrandId`